grails-app/conf/Config.groovy

```diff
@@ -257,18 +257,13 @@ portal {
 
         dataFile = [
             "WWW:DOWNLOAD-1.0-http--download",
-            "WWW:DOWNLOAD-1.0-http--downloaddata",
-            "WWW:DOWNLOAD-1.0-http--downloadother",
-            "WWW:LINK-1.0-http--downloaddata"
+            "IMOS:AGGREGATION--gogoduck",
+            "IMOS:AGGREGATION--bodaac",
+            "IMOS:AGGREGATION--aodaac"
```

Comment:
@danfruehauf 
They are referred to in the portal code to get the download links.
BaseInjector.js: 29
```
return dataCollection.getLinksByProtocol(Portal.app.appConfig.portal.metadataProtocols.dataFile);
```
I could be missing something here so let me know :)